pkg/networkmanager/interfaces.js

```diff
@@ -829,8 +875,9 @@ export function NetworkManagerModel() {
                 .finally(pop_refresh);
     }
 
-    function handle_updated(obj) {
-        refresh_settings(obj);
+    function handle_updated(obj, things) {
+        if (things.length > 0)
+            refresh_settings(obj);
```

Comment:
Oh, please don't do that. That was an experiment of mine, but this is in no way documented to actually get a list of "things". I expect this to break a *lot*, and indeed tests are wildly unhappy. Please revert.